src/atlas/util/VortexRollup.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * (C) Crown Copyright 2022 Met Office
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ * In applying this licence, ECMWF does not waive the privileges and immunities
+ * granted to it by virtue of its status as an intergovernmental organisation
+ * nor does it submit to any jurisdiction.
+ */
+
+#pragma once
+
+#include "atlas/util/Constants.h"
+#include "atlas/util/Earth.h"
```

Comment:
Please move these headers to VortexRollup.cc

Reply:
Done